main.go

```diff
@@ -87,7 +88,10 @@ func main() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	opts := zap.Options{
-		Development: true,
+		Development: false,
```

Comment:
Is this change on purpose?  Going from false -> true?

Reply:
It was on purpose since this would let the operator use production logging defaults. It could still be overridden when developing by using `--zap-devel=true`

https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/building-operators/golang/references/logging.md#default-zap-logger